README.md

```diff
@@ -44,15 +44,41 @@ To run checks against an assignment, use the `gator grade` command.
 
 #### Interpreting Output
 
-Output team
+A dictionary will come from the input team that mostly will contain list,
+and a dictionary. The dictionary will contain two keys (shell and gatorgrader) that will contain the commands for the output team to run, run through gatorgrade, and return as a list of tuples.
+Each tuple will contain a descriptive boolean that will either Pass or Fail.
+If Passed, it will come through as an empty string. If Failed it will come
+through as what exactly failed within the assignment.
 
 ### Generating a gatorgrade.yml file
 
-Generate team
+In order to generate a ‘gatorgrade.yml’ file, you should run the following
+command in your terminal:
+
+‘’’
+Gator grade generate
+‘’’
+
+Make sure that your repository doesn’t already contain a gator grade.yml file,
+or the command will not run. If you want to override this, run the previous
+command with —force/-f:
+
+‘’’
+Gator grade generate —force
+‘’’
+
+The created .yml file will be a template with example checks inside of it.
 
 #### Configuring GatorGrade Checks
 
-Input team
+In order to configure GatorGrade checks, you need to go into the
+'gatorgrade.yml' file and manually edit the file. You can input different
+keywords into the yml format, which is labeled for ease of use. For example,
+you can input check 'match file fragment' in order to make sure that a
+specific line of code is included in a program. You can also use the
+'match file fragment' check with the 'exact' option with value 0 in order to
+create a check that can make sure that there are no TODO markers left in a
+completed repository.
```

Comment:
Hi @burgess01! It was my understanding that these portions of the README.md would be completed by members of the other teams. Since this was announced in the #sep channel on Discord, I believe some teams have accounted for this task in their task planning for this sprint. Could someone from the Integration team clarify what portions should be completed by other teams and what portions will be handled by the Integration team?